options.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>BigConsole Options</title>
+	</head>
+	<body>
+		<label>
+			Editor on
+			<select id="layout">
+				<option value="right">Right</option>
+				<option value="left">Left</option>
+				<option value="top">Top</option>
+				<option value="bottom">Bottom</option>
+			</select>
+		</label>
+		<script src="./options.js"></script>
+	</body>
+</html>
```

Comment:
I think it would be nice to add a bit of styling to this page:

```css
html {
    background-color: #FAFAFA;
}
body {
    background-color: #EEEEEE;
    border: 1px solid #CCCCCC;
    margin: 40px auto;
    padding: 40px;
    width: 680px;
}
```

...and potentially a title in case people are like me and have so many tabs that they can't see the tab titles any more 😁